package.json

```diff
@@ -17,12 +17,14 @@
     "deploy:full": "forge script -vvv script/DeployTestnet.s.sol:DeployTestnet --rpc-url $SCRIPT_RPC_URL --private-key $SCRIPT_PRIVATE_KEY --sender $SCRIPT_SENDER --broadcast $SCRIPT_VERIFY $SCRIPT_ETHERSCAN --priority-gas-price $SCRIPT_PRIORITY_GAS_PRICE && npm run test:deployment",
     "----------CHAIN-COMMANDS----------": "",
     "deploy:local": "CONFIG=config/localhost.json SCRIPT_RPC_URL=http://localhost:8545 SCRIPT_PRIVATE_KEY=$LOCAL_PRIVATE_KEY SCRIPT_SENDER=$LOCAL_SENDER SCRIPT_PRIORITY_GAS_PRICE=100000 npm run deploy:full",
+    "deploy:localfork": "CONFIG=config/optimismSepolia.json SCRIPT_RPC_URL=http://localhost:8545 SCRIPT_PRIVATE_KEY=$LOCAL_PRIVATE_KEY SCRIPT_SENDER=$LOCAL_SENDER SCRIPT_PRIORITY_GAS_PRICE=100000 npm run deploy:full",
```

Comment:
This seems unnecessary since we can modify localhost.json and run `deploy:local`. Maybe we should add localhost.json to the gitignore so that testers can change it as needed without having to modify and revert changes to a tracked file.